app/scripts/views/sign_in_totp_code.js

```diff
@@ -4,14 +4,14 @@
 
 const AuthErrors = require('lib/auth-errors');
 const Cocktail = require('cocktail');
-const Constants = require('../lib/constants');
 const FormView = require('./form');
 const SignInMixin = require('./mixins/signin-mixin');
 const ServiceMixin = require('./mixins/service-mixin');
 const Template = require('templates/sign_in_totp_code.mustache');
 const VerificationReasonMixin = require('./mixins/verification-reason-mixin');
 
 const CODE_INPUT_SELECTOR = 'input.totp-code';
+const VALID_TOTP_CODE_REGEX = /^[0-9]{6}/;
```

Comment:
You'll need `$` on the end of this regexp too, otherwise the TOTP code will be a minimum of 6 chars, not exactly.